src/modality-specific-files/stimuli.md

```diff
@@ -87,11 +64,22 @@ A guide for using macros can be found at
 -->
 {{ MACROS___make_sidecar_table("stimulus.Stimulus") }}
 
-## Stimuli.tsv and Stimuli.json
+In some cases, such as observing the copyright of a stimulus file, the actual stimulus file may not be shared. In such cases, the `stim-<label>_<suffix>.json` file SHOULD be used to provide metadata about the stimulus file, including the license, copyright, URL, and description.
```

Comment:
Up above the underbar was removed before the suffix:  <suffix> not _<suffix>.  It should be consistent.

Reply:
Thanks, there was one from the tree with the miggning underbar. Overall, I used either *suffix* or `_<suffix>`, which hopeully have clear differences. Please feel free to modify.